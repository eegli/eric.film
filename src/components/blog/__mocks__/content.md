# !!! YOU ARE IN DEVELOPMENT MODE !!!

Der Bundesrat plant eine weitere Lockerung der Corona-Massnahmen. ğŸ‘€ So sollen Veranstaltungen bis 1000 Personen zugelassen werden. Auch die Polizeistunde soll fallen.

# H1 Header

## H2 Header

### H3 Header

#### H4 Header

# First, let me show you how I write code

This is the Blog Content and it's a code block

```jsx
const BlogContent = ({ title, excerpt, content }) => {
  return (
    <BlogPostContainer>
      <BlogPostTitle>{title}</BlogPostTitle>
      <BlogPostExcerpt>{excerpt}</BlogPostExcerpt>
      <StyledMD
        renderers={{
          code: CodeBlock,
        }}
        source={content}
      />
    </BlogPostContainer>
  );
};
```

For this, I use a package called `react-markdown`. Can you see how this was inline-code?

# What about a table?

Text

# Here is a unordered list and this is a h1

This list has content. Das hat der Bundesrat beschlossen:

- Ab kommenden Montag ist fast alles so wie frÃ¼her. Der Bundesrat hebt angesichts der guten Entwicklung fast alle Corona-Massnahmen auf.
- Veranstaltungen bis 1000 Personen sind ab Montag wieder erlaubt. Das Verbot fÃ¼r Events mit Ã¼ber 1000 Teilnehmern bleibt wie schon seit LÃ¤ngerem angekÃ¼ndigt noch bis Ende August bestehen.
- Die Abstandsregel wird gelockert. Neu empfiehlt der Bundesrat nicht mehr einen Abstand von 2, sondern 1,5 Metern. Die Regierung hat ausserdem die Vorgaben fÃ¼r die Schutzkonzepte vereinfacht und vereinheitlicht. Es gelten jetzt fÃ¼r alle die gleichen Richtlinien.

# What about a checklist?

- [x] To be done
- [ ] To be completed
- [x] Not done!

# Here is a numbered list and this is a h1

1. This list has content. Das hat der Bundesrat beschlossen:
2. Ab kommenden Montag ist fast alles so wie frÃ¼her. Der Bundesrat hebt angesichts der guten Entwicklung fast alle Corona-Massnahmen auf.
3. Veranstaltungen bis 1000 Personen sind ab Montag wieder erlaubt. Das Verbot fÃ¼r Events mit Ã¼ber 1000 Teilnehmern bleibt wie schon seit LÃ¤ngerem angekÃ¼ndigt noch bis Ende August bestehen.
4. Die Abstandsregel wird gelockert. Neu empfiehlt der Bundesrat nicht mehr einen Abstand von 2, sondern 1,5 Metern. Die Regierung hat ausserdem die Vorgaben fÃ¼r die Schutzkonzepte vereinfacht und vereinheitlicht. Es gelten jetzt fÃ¼r alle die gleichen Richtlinien.

# This section has different text styles applied to it and this is a h1

Ãœberall habe ein **Lernprozess** stattgefunden, auch bei den _Kantonen_, sagt BundesprÃ¤sidentin Sommaruga auf die Frage, ob sie keine Angst hat, dass die Kantone Ã¼berfordert sein werden. Â«_Dieser Lernprozess hilft jetzt, Entscheidungen zu fÃ¤llen.Â» Sie gibt sich zuversichtlich._ Â«Jetzt mit 10 bis 35 Ansteckungen pro Tag macht es Sinn, gezielt einzugreifen.Â» Der Bund ~~verschwinde aber nicht einfach~~, man habe ein Monitoring und sei in einer Â«viel besseren AusgangslageÂ».

# This section will be a citation and is a h1

Gleichzeitig mit den Lockerungen wird auch die Homeoffice-Empfehlung gelockert, auch fÃ¼r die besonders gefÃ¤hrdeten Gruppen. Die seien nicht weniger gut geschÃ¼tzt, sagt BundesprÃ¤sidentin Sommaruga. Die Arbeitgeber hÃ¤tten unverÃ¤ndert eine FÃ¼rsorge.

> Â«Da hat sich nichts geÃ¤ndert.Â»

Berset ergÃ¤nzt, dass der Abstand oder die Maske fÃ¼r alle gelte â€“ also auch am Arbeitsplatz.

# Abstand halten, Masken oder andere Verhaltensregeln und ein Link...

...werden erst fallen, wenn es eine Impfung oder eine gute Behandlung gibt, sagt Alain Berset. Auch die 1,5 Meter Abstandsregeln sei notwendig. 1 Meter sei schon normal und es brauche einen zusÃ¤tzlichen Abstand als im Normalfall. [Ich arbeite beim Blick](https://www.blick.ch). Das ist eine Newsagentur.

# Here is an image

![6288344351929447784604DJI_0017-1024x767.jpg](https://media.graphcms.com/xeBuhftsQPS4AxuxewLU)
And last but not least, **phatt text!**
