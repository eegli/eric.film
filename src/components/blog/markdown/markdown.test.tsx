import { render } from '@/src/tests/testUtils';
import { blogPostContent } from '@/src/tests/__mocks__';
import Markdown from './markdown.component';

describe('Markdown', () => {
  it('renders markdown content', () => {
    const { asFragment } = render(<Markdown {...blogPostContent} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
