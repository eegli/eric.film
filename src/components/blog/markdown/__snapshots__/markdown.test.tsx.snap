// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Markdown renders markdown content 1`] = `
<DocumentFragment>
  .c0 {
  margin-top: 1rem;
  background-color: #222222;
  line-height: 1.8rem;
  color: #f7f7f7;
}

.c0 > * {
  padding: 0 1rem;
}

.c0 img,
.c0 code {
  padding: 0;
}

.c0 li {
  margin: 0 1rem;
}

.c0 p {
  font-size: 16px;
}

.c0 h1 {
  color: #feb8ce;
}

.c0 code {
  color: #feb8ce;
}

.c0 a {
  color: #f7f7f7;
  -webkit-text-decoration: underline;
  text-decoration: underline;
  cursor: pointer;
}

.c0 img {
  max-width: 100%;
  max-height: 100%;
  display: block;
  margin: 0.5rem auto;
}

.c0 blockquote {
  background: #3b3b3b;
  border-left: 4px solid #f7f7f7;
  margin: 1.5rem 2rem;
  padding: 0.5em 10px;
}

.c0 blockquote > p {
  display: inline;
  color: #F1F1F1;
  font-style: italic;
}

.c0 blockquote:before {
  color: #ccc;
  content: open-quote;
  font-size: 2em;
  line-height: 0.1em;
  margin-right: 0.25em;
  vertical-align: -0.4em;
}

<div
    class="c0"
  >
    <p>
      Fin oui eut sillons oui fausser encourt station souffle.Fin oui eut sillons oui fausser encourt station souffle.Fin oui eut sillons oui fausser encourt station souffle.Fin oui eut sillons oui fausser encourt station souffle.Fin oui eut sillons oui fausser encourt station souffle.Fin oui eut sillons oui fausser encourt station souffle.Fin oui eut sillons oui fausser encourt station souffle
    </p>
  </div>
</DocumentFragment>
`;
